### DISCUSSION TITLE
input tt-hint

### DISCUSSION THREAD

**Discussion by speich** [digitalfortress-tech/typeahead-standalone#21] [Q&A] [answered]
What is the purpose of the second input field with class tt-hint?

**Comment by niketpathak** [comment, marked as answer]
Hello Simon, 

A second input field (read-only) is used to display a hint within the original input field. For example, head to https://typeahead.digitalfortress.tech/, and try typing "g" in the first input box that you see. As soon as you do that, you should see 3 suggestions and within the input field, you will also see the letters _old_ which are a bit greyed out, like a placeholder. This placeholder value comes from the second input field with class `tt-hint`. If you hit the <kbd>Tab</kbd> key, it autocompletes your input with the hint value. Using this hint is optional and you can disable it 

```javascript
typeahead({
    input: ...,
    source: {
      ...
    },
    hint: false,   // disables the 2nd input field
});
```

Hope that answers your question. 

Good day!

**Reply by speich** [reply]
Thanks for the detailed explanation. Good day to you, too.
